Example_Programs/programming_concpets/calculator_complex/test/test_complex_calculator.c

```diff
@@ -46,11 +72,23 @@ void test_negative(void)
     
     c2.real = 75;
     c2.imaginary = 95;
-    
+
     TEST_ASSERT_EQUAL(SUCCESS, complex_sum(&c1, &c2, &result));
     TEST_ASSERT_EQUAL(50, result.real);
     TEST_ASSERT_EQUAL(80, result.imaginary);
 
+    TEST_ASSERT_EQUAL(SUCCESS, complex_diff(&c1, &c2, &result));

```

Comment:
Write different test cases for each function written. It is more readable and also easy to find the bugs.

Reply:
Sure sir. I will do it.